packages/pango.rb

```diff
@@ -23,7 +23,13 @@ def self.build
     ENV['CFLAGS'] = "-fuse-ld=lld"
     ENV['CXXFLAGS'] = "-fuse-ld=lld"
     system "meson",
-      #{CREW_MESON_OPTIONS},
+      "-Dprefix=#{CREW_PREFIX}",
+      "-Dlibdir=#{CREW_LIB_PREFIX}",
+      "-DLIB_INSTALL_DIR=#{CREW_LIB_PREFIX}",
+      "-Dmandir=#{CREW_MAN_PREFIX}",
+      "-DSYSCONFDIR=#{CREW_PREFIX}/etc",
+      "-Ddatadir=#{CREW_LIB_PREFIX}",
+      '-Dbuildtype=release',
       "-Dinstall-tests=false",
       "builddir"
```

Comment:
Try this:
```
system "meson #{CREW_MESON_OPTIONS} -Dinstall-tests=false builddir"
```

Reply:
We can try again after the sanitized meson options in https://github.com/skycocker/chromebrew/pull/4515